### ISSUE TITLE
34872220: iPhone X statusbar elements lack accessibility declarations

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#18591]
#### Description

Summary:
Elements in the statusbar of the iPhone X are not correctly declared according to accessibility and lack identifiers, traits and types. Even the statusbar itself is not declared as statusbar.

In previous iOS versions and on other iPhones the statusbar elements already lack some declarations, but at least some are set.

Apart from being accessible, it is important, that those elements use accessibility correctly to be able to assert usage of location services or network activity after specific actions have been taken in XCUITests.

Steps to Reproduce:
1. Create a single view application including UI tests
2. Try to find specific elements, e.g. the network activity indicator or the geolocation indicator using XCUIElementQueries like `XCUIApplication().statusBars.activityIndicators[“NetworkActivityIndicator”]` or `XCUIApplication().statusBars.activityIndicators[“GeofenceLocationTrackingOn”]`

Expected Results:
I expect that elements returned by the above query have correct types, e.g. “activityIndicator” or “statusItem”; and I expect them to have identifiers, since they are unique: there is just one network activity indicator, just one geolocation indicator and just one clock in the statusbar.

Actual Results:
On the iPhone X the only element that is not of type “other” and has any accessibility attributes set is the clock: “StaticText, 0x608000383dc0, traits: 64, {{29.3, 14.0}, {37.3, 18.0}}, label: '9:33 ‘”. All other elements are of type “other” and have no accessibility attributes set at all. Even the statusbar on the iPhone X is not declared as “statusBar” but as type “other”.

On iPhones with a regular statusbar, the statusbar is correctly declared as “statusBar” and some elements, e.g. the geolocation indicator has an identifier set. But for these iPhones as well none of the elements has a correct type and most elements don’t have identifiers, just labels.

Version:
11.0

Notes:
I attached a zip file containing two txt files that contain the output of the `debugPrint(XCUIApplication())` command during a UI test in an empty single view application. In these outputs you can clearly see the lack of accessibility declarations.

-
Product Version: 11.0
Created: 2017-10-07T16:06:36.694350
Originated: 2017-10-07T18:06:00
Open Radar Link: http://www.openradar.me/34872220
